server/chat-commands/moderation.ts

```diff
@@ -1191,11 +1236,16 @@ exports.commands = {
 		if (!target) return this.parse('/help htmldeclare');
 		if (!this.can('gdeclare', null, room)) return false;
 		if (!this.canTalk()) return;
-		target = this.canHTML(target);
+		(target as string | null) = this.canHTML(target);
 		if (!target) return;
 
-		for (let u in room.users) {
-			if (Users.get(u).connected) Users.get(u).sendTo(room, `|notify|${room.title} announcement!|${Chat.stripHTML(target)}`);
+		for (const u in room.users) {
+			if (Users.get(u)?.connected) {
+				Users.get(u)?.sendTo(
+					room,
+					`|notify|${room.title} announcement!|${Chat.stripHTML(target)}`
+				);
+			}
```

Comment:
```suggestion
			Users.get(u)?.sendTo(
				room,
				`|notify|${room.title} announcement!|${Chat.stripHTML(target)}`
			);
```

There's no need for the `connected` check, tbh.